include/rocksdb/options.h

```diff
@@ -331,6 +331,11 @@ struct ColumnFamilyOptions : public AdvancedColumnFamilyOptions {
   // Default: nullptr
   std::shared_ptr<SstPartitionerFactory> sst_partitioner_factory = nullptr;
 
+  // Automatic flush after tombstone count in memtable hits this limit.
+  // helps with delete-range + read workloads
+  //  0 to disable it completely
+  uint32_t max_tombstones_count = 0;
```

Comment:
Can we use range_tombstone/range_deletion to disambiguate from point tombstones? Also maybe include memtable in the name like memtable_max_range_deletion.

Reply:
Yes, that makes sense, i'll do that change.